insights/client/phase/v1.py

```diff
@@ -20,7 +20,11 @@
 def phase(func):
     @functools.wraps(func)
     def _f():
-        compile_config()
+        try:
+            compile_config()
+        except ValueError as e:
+            print('ERROR:' + e)
```

Comment:
looks like other messages are printed to stderr via logging.

Reply:
oh shoot.  I see that logging may not be set up at this point.  carry on.

Reply:
Actually, thinking about it even more, I believe the right thing to do is to print this message to stderr.  Since logging isn't guaranteed to be set up, you probably need to do `sys.stderr.write('ERROR: ' + e + '\n')`